modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -295,9 +297,13 @@ public Object getWorkOwner() {
 
   @Override
   public void setListener(Processor listener) {
-    messageProcessor = listener;
+    // Ignore, this source only needs AsyncProcessor listener
```

Comment:
this should not ignore. This method should ideally not exist or at most throw UnsupportedOperationException